lib/DDG/Spice/Forecast.pm

```diff
@@ -12,28 +12,30 @@ secondary_example_queries "weather 12180";
 topics "everyday", "travel";
 code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Forecast.pm";
 
-my @temperatures = qw(temp temperature);
-my @connectors   = qw(in for at);
-my @descriptors  = qw(current);
-
-my @temps_triggers;
-
-foreach my $temp (@temperatures) {
-    foreach my $conn (@connectors) {
-        push @temps_triggers, $temp . ' ' . $conn;    # temperature for, temp at
-        foreach my $desc (@descriptors) {
-            push @temps_triggers, join(' ', $desc, $temp, $conn), $desc . ' ' . $temp;    # current temperature in; current temp
-        }
-    }
-}
-
-my @forecast_words = qw(forecast forcast weather);
-my @triggers = (@forecast_words, @temps_triggers, 'meteo');
+# my @temperatures = qw(temp temperature);
+# my @connectors   = qw(in for at);
+# my @descriptors  = qw(current);
+# 
+# my @temps_triggers;
+# 
+# foreach my $temp (@temperatures) {
+#     foreach my $conn (@connectors) {
+#         push @temps_triggers, $temp . ' ' . $conn;    # temperature for, temp at
+#         foreach my $desc (@descriptors) {
+#             push @temps_triggers, join(' ', $desc, $temp, $conn), $desc . ' ' . $temp;    # current temperature in; current temp
+#         }
+#     }
+# }
+
+my @forecast_words = qw(weather);
+my @triggers = (@forecast_words);
 triggers startend => @triggers;
 
 spice from => '([^/]*)/?([^/]*)';
 spice to => 'http://forecast.io/ddg?apikey={{ENV{DDG_SPICE_FORECAST_APIKEY}}}&q=$1&callback={{callback}}';
 
+# skip synonyms for "weather" because of overtriggering
+my @weather_synonyms =qw(forecast);
```

Comment:
Haha...well, since we're still using proper noun Forecast as the provider, let's leave it for now :)